spec/factories/tasks.rb

```diff
@@ -5,5 +5,10 @@
     from = Date.parse("2019/08/01")
     to   = Date.parse("2019/12/31")
     deadline { Random.rand(from..to) }
+
+    trait :done do
+      status {:done}
+      completion_date {Time.current.yesterday}
```

Comment:
波括弧（ブレース）の間は半角スペースで開けた方が見やすいかと！
```suggestion
      status { :done }
      completion_date { Time.current.yesterday }
```